homeassistant/const.py

```diff
@@ -429,10 +429,10 @@
 # Degree units
 DEGREE: Final = "°"
 
-# Currency units
-CURRENCY_EURO: Final = "€"
-CURRENCY_DOLLAR: Final = "$"
-CURRENCY_CENT: Final = "¢"
+# Currency units, any ISO4217 currency code is also allowed
```

Comment:
Maybe say that only the ISO units are allowed, besides default currency?

Reply:
Sounds good.
Should we drop the `CURRENCY_US_DOLLAR` constant, it's only used by the isy994 integration?

Reply:
We can drop it here for now but we can move it to that integration to still have a constant there.